proof/ROOT

```diff
@@ -73,17 +73,11 @@ session AInvs in "invariant-abstract" = ASpec +
   directories
     "$L4V_ARCH"
   theories [condition = "SKIP_AINVS_PROOFS", quick_and_dirty, skip_proofs]
-    "KernelInit_AI"
-    "$L4V_ARCH/ArchDetSchedSchedule_AI"
-    "$L4V_ARCH/ArchFPU_AI"
+    "AInvsToplevel_AI"
```

Comment:
I like the new top-level theory. Having to click through all the single theories has always been annoying and a risk for forgetting one interactively.

Reply:
Thanks for doing this. We now have an AI-enhanced toplevel theory :)